params/params.yaml

```diff
@@ -0,0 +1,319 @@
+    # nav2_bt_navigator/navigate_through_poses_w_replanning_and_recovery.xml
+    # They can be set here or via a RewrittenYaml remap from a parent launch file to Nav2.
+
+    # plugin_lib_names is used to add custom BT plugins to the executor (vector of strings).
+    # Built-in plugins are added automatically
+    # plugin_lib_names: []
+
+    error_code_names:
+      - compute_path_error_code
+      - follow_path_error_code
+
+controller_server:
+  ros__parameters:
+    #     controller_frequency: 20.0
+    #     min_x_velocity_threshold: 0.001
+    #     min_y_velocity_threshold: 0.5
+    #     min_theta_velocity_threshold: 0.001
+    #     failure_tolerance: 0.3
+    #     progress_checker_plugins: ["progress_checker"]
+    #     goal_checker_plugins: ["general_goal_checker"] # "precise_goal_checker"
+    controller_plugins: ["FollowPath"]
+    #     use_realtime_priority: false
+
+    #     # Progress checker parameters
+    #     progress_checker:
+    #       plugin: "nav2_controller::SimpleProgressChecker"
+    #       required_movement_radius: 0.5
+    #       movement_time_allowance: 10.0
+    #     # Goal checker parameters
+    #     #precise_goal_checker:
+    #     #  plugin: "nav2_controller::SimpleGoalChecker"
+    #     #  xy_goal_tolerance: 0.25
+    #     #  yaw_goal_tolerance: 0.25
+    #     #  stateful: True
+    #     general_goal_checker:
+    #       stateful: True
+    #       plugin: "nav2_controller::SimpleGoalChecker"
+    #       xy_goal_tolerance: 0.25
+    #       yaw_goal_tolerance: 0.25
+    FollowPath:
+      plugin: "nav2_regulated_pure_pursuit_controller::RegulatedPurePursuitController"
+      desired_linear_vel: 0.5
+      lookahead_dist: 0.6
+      min_lookahead_dist: 0.3
+      max_lookahead_dist: 0.9
+      lookahead_time: 1.5
+      rotate_to_heading_angular_vel: 1.8
+      transform_tolerance: 0.1
+      use_velocity_scaled_lookahead_dist: false
+      min_approach_linear_velocity: 0.05
+      approach_velocity_scaling_dist: 1.0
+      use_collision_detection: true
+      max_allowed_time_to_collision_up_to_carrot: 1.0
+      use_regulated_linear_velocity_scaling: true
+      use_cost_regulated_linear_velocity_scaling: false
+      regulated_linear_scaling_min_radius: 0.9
+      regulated_linear_scaling_min_speed: 0.25
+      use_fixed_curvature_lookahead: false
+      curvature_lookahead_dist: 1.0
+      use_rotate_to_heading: true
+      rotate_to_heading_min_angle: 0.785
+      max_angular_accel: 3.2
+      max_robot_pose_search_dist: 10.0
+      interpolate_curvature_after_goal: false
+      cost_scaling_dist: 0.3
+      cost_scaling_gain: 1.0
+      inflation_cost_scaling_factor: 3.0
+
+local_costmap:
+  local_costmap:
+    ros__parameters:
+      update_frequency: 5.0
+      publish_frequency: 2.0
+      global_frame: odom
+      robot_base_frame: base_link
+      rolling_widow: true
+      width: 3
+      height: 3
+      resolution: 0.05
+      robot_radius: 0.22
```

Comment:
truck radius should be measured and added

Reply:
Pushed to a new story: [CF-823](https://usdot-carma.atlassian.net/browse/CF-823)